src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {

```

Comment:
![image](https://github.com/user-attachments/assets/e2144b06-0a42-475c-ae75-7aaa0f73a44b)
l to L